pkg/config/config.go

```diff
@@ -277,7 +277,7 @@ type LibpodConfig struct {
 	RuntimeSupportsNoCgroups []string `toml:"runtime_supports_nocgroups"`
 
 	// SetOptions contains a subset of config options. It's used to indicate if
-	// a given option has either been set by the user or by a parsed libpod
+	// a given option has either been set by the user or by a parsed engine
```

Comment:
I'm on the fence about "engine" vs " the engine" in spots like this in the internal doc.  Unfortunately engine is fine in many places, but in others, I think it could use a "the" in front of it.   Your call, I'm good either way.

Reply:
Removed engine and changed to `the parsed configuration file.`